matchms/Spectrum.py

```diff
@@ -33,6 +33,14 @@ class Spectrum:
         0.7
         spectrum1
 
+    To visually inspect a spectrum run ``spectrum.plot()``
+
+    .. figure:: paper/spectrum-plot-example.png
```

Comment:
Picture does not show, can you copy it to `readthedocs/_static/` directory

Reply:
Would it then be called by ``.. figure:: readthedocs/_static/spectrum-plot-example.png`` ?

Reply:
I think so

Reply:
I now added both the workflow and the spectrum.plot() example figure to the ``_static`` folder... let's see if that works.